src/lib/isPostalCode.js

```diff
@@ -5,6 +5,14 @@ const threeDigit = /^\d{3}$/;
 const fourDigit = /^\d{4}$/;
 const fiveDigit = /^\d{5}$/;
 const sixDigit = /^\d{6}$/;
+const validators = {
+  ES: function ES(str) {
+    // sanitize user input
+    let sanitizedInput = str.trim();
+    return sanitizedInput.length === 5 && parseInt(sanitizedInput, 10) >= 1001
```

Comment:
Can we use a pattern instead, just like the rest?

Reply:
sure